web/src/app/core/resolvers/edit-project-token.resolver.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, Resolve, Router } from '@angular/router';
+import { of } from 'rxjs';
+import { catchError, take } from 'rxjs/operators';
+
+// Dashboard hub model and services
+import { ProjectTokenModel } from '../../shared/models/index.model';
+import { TokenService } from '../services/token.service';
```

Comment:
can we add token.service to index file ? and import from index file similar to model?

Reply:
I renamed 'token.service' to 'project-token.service'.
sure, you can add service and modal to index files